modules/nw-egressnetworkpolicy-create.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift-sdn/configuring-egress-firewall.adoc
+
+[id="nw-networkpolicy-create_{context}"]
+= Creating an egress firewall policy object
+
+As a {product-title} cluster administrator, you can create an egress firewall policy object for a project.
+
+[IMPORTANT]
+====
+If the project already has an EgressNetworkPolicy object defined, you must edit the existing policy to make changes to the egress firewall rules.
+====
+
+.Prerequisites
+
+* A cluster using the OpenShift SDN network provider plug-in.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster as a cluster administrator.
+
+.Procedure
+
+. Create a policy rule:
+.. Create a `<policy-name>.yaml` file where `<policy-name>` describes the egress
+policy rules.
+.. In the file you just created define an egress policy object.
+
+. Enter the following command to create the policy object:
++
+----
+$ oc create -f <policy-name>.yaml -n <project>
+----
++
+In the following example, a new EgressNetworkPolicy object is created in a
+project named `project1`:
++
+----
+$ oc create -f default-rules.yaml -n project1
+egressnetworkpolicy.network.openshift.io/default-rules created
+----
++
+. Optionally, save the `<policy-name>.yaml` so that you can make changes later.
```

Comment:
s/Optionally,/Optional:

> If a step is optional, begin the step with the word Optional. If your authoring tool provides a
semantic element to generate the word Optional, use that element.
Example
```
1. Optional: Click Set password. In the Password field, type a password.
```